packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -188,7 +198,8 @@ export class ArtifactCreator {
 
   async captureScreenshot(
       renderer: string, scenarioName: string, dimensions: Dimensions,
-      outputPath: string = join(this.outputDirectory, 'model-viewer.png')) {
+      outputPath: string = join(this.outputDirectory, 'model-viewer.png'),
+      useMaxTime: boolean = false, maxTimeInSec: number = 60) {
```

Comment:
Perhaps instead of adding a `useMaxTime` flag, specifying a `maxTimeInSec` of 0 could denote no timeout.

Reply:
Or even -1, to make it even more obvious. It's a good practice to test that inputs are >0 anyway if that's what you expect.